test/unit/react-imgix.test.jsx

```diff
@@ -237,19 +237,65 @@ describe("When in <source> mode", () => {
     it("props.srcSet should be set to a valid src", () => {
       expect(renderImage().props().srcSet).toContain(src);
     });
-    it("srcSet should be in the form src, src 2x, src 3x, src 4x, src 5x", () => {
+
+    it("should have a srcSet set correctly", async () => {
+      const srcset = renderImage().props().srcSet;
+      expect(srcset).not.toBeUndefined();
+      expect(srcset.split(", ")[0].split(" ")).toHaveLength(2);
+      const aSrcFromSrcSet = srcset.split(", ")[0].split(" ")[0];
+      expect(aSrcFromSrcSet).toContain(src);
+      const aWidthFromSrcSet = srcset.split(", ")[0].split(" ")[1];
+      expect(aWidthFromSrcSet).toMatch(/^\d+w$/);
+    });
+
+    it("returns the expected number of `url widthDescriptor` pairs", function() {
+      const srcset = renderImage().props().srcSet;
+
+      expect(srcset.split(",").length).toEqual(targetWidths.length);
+    });
+
+    it("should not exceed the bounds of [100, 8192]", () => {
+      const srcset = renderImage().props().srcSet;
+
+      const srcsetWidths = srcset
+        .split(", ")
+        .map(srcset => srcset.split(" ")[1])
+        .map(width => width.slice(0, -1))
+        .map(Number.parseFloat);
+
+      const min = Math.min(...srcsetWidths);
+      const max = Math.max(...srcsetWidths);
+
+      expect(min).not.toBeLessThan(100);
+      expect(min).not.toBeGreaterThan(8192);
```

Comment:
@jayeb I assume this should be `max`? It appeared this same way elsewhere for a different test. Also, is this still the maximum width we want to enforce?

Reply:
Yea, seems like it should be `max`. And yes, this is still the maximum width our painters will render so it makes sense to enforce it as the maximum image size in the srcset.

Reply:
Ah interesting because the largest width generated is currently 8584, but I think I see how that slipped through.